schemas/objects/sellOrderPrice.js

```diff
@@ -0,0 +1,20 @@
+export default {
+  title: 'Sell Order Price',
+  name: 'sellOrderPrice',
+  type: 'object',
+  fields: [
+    {
+      title: 'Sell order id',
+      name: 'sellOrderId',
+      type: 'string',
+      validation: Rule => Rule.required(),
+    },
+    {
+      title: 'Sell order price',
+      description: 'price per credit in USD',
+      name: 'usdPrice',
```

Comment:
this gives us more flexibility to put the price at the sell order level but if we think that we'll want to always define the same credit price for a given project, we could put this at the project document level cc/ @clevinson

Reply:
Makes sense to me.